docs/UserGuide.md

```diff
@@ -330,7 +331,7 @@ Examples:
 * Group formed cannot be deleted individually unless using the [`group` command](#771-forming-focus-groups-manually--group) to clear all groups <br>
   eg. `group id/A0123459X` removes student with `STUDENT_ID = A0123459X` from all the groups the student was in.
 * Random groups formed will not be affected by the [`setweak` command](#714-setting-weak-threshold--setweak)<br>
-  ie. student in a randomly formed group will remain in the group after `setweak` command, even if the student falls above the new weak threshold
+  *  i.e. students in a randomly formed group will remain in the group after `setweak` command, even if the student falls above the new weak threshold
```

Comment:
Any reason why the bullet point is needed since line 332 does not have one.

Reply:
I did that because the sentence is pretty long. I will replace it with brackets then.

Reply:
Do you think the current formatting looks fine, else it would be inconsistent with the previous line
![image](https://github.com/AY2324S2-CS2103T-T09-1/tp/assets/108446221/769ccca7-44e6-4b5f-869b-949c55a94635)

Reply:
Fair point, I will remove the bracket. So it will look like the picture above.